Gemfile~

```diff
@@ -2,4 +2,8 @@
 # A sample Gemfile
 source "https://rubygems.org"
 
+# web framework
 gem "sinatra"
+
+# tests
+gem "rspec"
```

Comment:
This is a temporary file (tilde character at the end). You should add file to global `.gitignore`.